bin/build-odroid_c2-image.sh

```diff
@@ -37,5 +37,4 @@ umount $TARGET/proc
 umount $TARGET
 losetup -d /dev/loop0
 
-mkdir -p $OUTPUT
-pv < $TEMPDIR/odroid-c2.img | xz -0 > $OUTPUT/odroid-c2-liquid.img.xz
```

Comment:
Good point, this might break the nightlies. I made a pull request in our infrastructure repo to update the script.